app/components/questionnaire/answerable-field.js

```diff
@@ -0,0 +1,58 @@
+import Ember from 'ember';
+
+
+const AnswerableField = Ember.Component.extend({
+  /**
+   * A base class for a simple answerable field, like a text field or radio choice
+   */
+  tagName: 'div',
```

Comment:
I thought `div` was the default tagName for Ember.
I haven't been including it in components.

Reply:
It is the default. Looks like I had set it in string-field.js previously then moved most of that code here.